typings/index.d.ts

```diff
@@ -864,6 +867,46 @@ declare module "noblox.js" {
         awardingUniverse: BadgeUniverse
     }
 
+    //Inventory
+
+    interface AssetOwner
+    {
+        userId: number;
+        username: string;
+        buildersClubMembershipType: number;
+    }
+    
+    interface CollectibleEntry {
+        userAssetId: number;
+        serialNumber?: number;
+        assetId: number;
+        name: string;
+        recentAveragePrice: string;
```

Comment:
Recent average price is a number, not a string.
According to test with my user id, `66592931`